client/utils/tags.tsx

```diff
@@ -95,7 +95,7 @@ export function injectConditionalTags(
   }
 
   // inject cap and Chinese gap for production Chinese
-  if (parsedHomeUrl.subdomain === 'chinese' && parsedHomeUrl.tld === 'org') {
+  if (env.clientLocale === 'chinese' && parsedHomeUrl.tld === 'org') {
```

Comment:
```suggestion
  if (homeLocation.split('/')[1] === 'chinese' && parsedHomeUrl.tld === 'org') {
```

Maybe something like this so we don't break the tests?